client/src/pages/Signup.jsx

```diff
@@ -112,12 +124,24 @@ function useRegister() {
   const login = async (username, email, password) => {
     console.log(email, password);
     const res = await fetch(
-      `/auth/signup?username=${username}&email=${email}&password=${password}`
-    ).then(res => res.json());
+      `/auth/signup?username=${username}&email=${email}&password=${password}`,
+      {
+        method: "POST",
+        // headers: { "Content-Type": "application/json" },
+        // body: { "email": email, "username": username, "password": password }
+      }
+    ).then((res) => res.json());
     console.log(res);
+    console.log(res.data.signupSuccess);
```

Comment:
We'll want to remove debugging console.logs from the code before committing it.

Reply:
Cleaned all code for the new PR.